.github/workflows/golangci-lint.yml

```diff
@@ -21,6 +21,8 @@ jobs:
         uses: actions/setup-go@v2
         with:
           go-version: 1.18.x
+      - name: Install snmp_exporter/generator dependencies
+        run: sudo apt-get update && sudo apt-get -y install libsnmp-dev
```

Comment:
```suggestion
        run: sudo apt-get update && sudo apt-get -y install libsnmp-dev
        if: github.repository == 'prometheus/snmp_exporter'
```

Reply:
Ohh, neat, I didn't know about that feature.